addons/google_calendar/models/google_calendar.py

```diff
@@ -253,7 +248,7 @@ def generate_data(self, cr, uid, event, isCreating=False, context=None):
         if not event.active:
             data["state"] = "cancelled"
 
-        if not self.get_need_synchro_attendee(cr, uid, context=context):
+        if not self.get_need_synchro_attendee():
             data.pop("attendees")
         if isCreating:
             other_google_ids = [other_att.google_internal_event_id for other_att in event.attendee_ids if other_att.google_internal_event_id]
```

Comment:
change variable name... google_internal_event_id is char field . required to change field name also...;)